sympy/functions/special/delta_functions.py

```diff
@@ -193,6 +198,12 @@ class is about to be instantiated and it returns either some simplified
         >>> DiracDelta(x - 100).subs(x, 100)
         DiracDelta(0)
 
+        Parameters
+        ==========
+
+        cls: DiracDelta class
```

Comment:
We don't need to list the cls argument for a classmethod.

Reply:
I will remove it then.